contracts/fee-manager/rustfmt.toml

```diff
@@ -0,0 +1,15 @@
+# stable
```

Comment:
Do we want different formatting options than the default? If not, lets delete this file.